packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCardPromotionCard.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+    <card-case
+        :card="card"
+        :class="[$style['c-stampCardPromotion1']]"
+        :data-test-id="testId">
+        <div
+            :class="[{ [$style['c-stampCardPromotion1-bgImg']]: !!card.image }]">
+            <img
+                :class="$style['c-stampCardPromotion1-img']"
+                :src="card.image"
+                alt=""> <!-- Alt-text not relevant for image - would repeat title text for screenreaders -->
+        </div>
+        <div :class="[$style['c-stampCardPromotion1-info']]">
+            <img
+                v-if="card.icon"
+                :class="[$style['c-stampCardPromotion1-icon']]"
+                :src="card.icon"
+                :data-test-id="testIdForSection('icon')"
+                alt=""> <!-- Alt-text not relevant for icon - would repeat title text for screenreaders -->
+            <div :class="$style['c-stampCardPromotion1-body']">
+                <h3
+                    v-make-text-accessible
+                    :class="[$style['c-stampCardPromotion1-title']]">
+                    {{ card.title }}
+                </h3>
+                <p
+                    v-make-text-accessible
+                    :class="[$style['c-stampCardPromotion1-statusText']]"
+                >
+                    {{ card.subtitle }}
+                </p>
+                <p
+                    v-if="card.url"
+                    :class="[
+                        'o-link--full',
+                        'o-link--bold',
+                        'u-color-link',
+                        'u-text-left',
+                        $style['c-stampCardPromotion1-link']
+                    ]"
+                    :data-test-id="testIdForSection('link')">
+                    {{ card.ctaText }}
+                </p>
+            </div>
+        </div>
+    </card-case>
+</template>
+
+<script>
+import CardCase from './CardCase.vue';
+import makeTextAccessible from '../MakeTextAccessible';
+
+export default {
+    name: 'StampCardPromotionCard',
+
+    components: {
+        CardCase
+    },
+
+    directives: {
+        makeTextAccessible
+    },
+
+    props: {
+        card: {
+            type: Object,
+            required: true
+        },
+
+        testId: {
+            type: String,
+            default: null
```

Comment:
is this null for a particular reason? Instead of just `''` which is falsey too.

Reply:
To be compliant to the docs I was going off [this description](https://vuejs.org/v2/guide/syntax.html#Attributes):

> If isButtonDisabled has the value of `null`, `undefined`, or `false`, the disabled attribute will not even be included in the rendered <button> element.

Which mentions `false` but not necessarily falsey values.

Reply:
I think it's probably safe to use the empty string as strictly it counts as one of Javascripts false values. Happy to leave as is tho.

Reply:
I think that semantically `null` makes more sense here.

Reply:
I'm inclined in 99% of the cases to go for stricter comparisons (i.e. `true` and `false` rather than `truthy` and `falsey`). I think `null` makes more sense here.